bios/dan.md

```diff
@@ -1,13 +1,13 @@
-# Jen Reiher
-![Picture of Jen](https://www.canadalearningcode.ca/wp-content/uploads/2017/07/jen.jpg)
+# Dan Clayton
+![Picture of Jen](https://www.facebook.com/photo.php?fbid=10155634302960333&l=956cd45eaf)
```

Comment:
we should change this to ![Picture of Dan] 
so that when you hover over it it says the right name